res/linux/mixxx.appdata.xml

```diff
@@ -36,6 +36,25 @@
       <li>Skinnable interface with several skins bundled</li>
     </ul>
   </description>
+  <provides>
+    <modalias>usb:v22F0p0008d*</modalias>
```

Comment:
If these IDs should match those in mixxx-usb-uaccess.rules we should add a comment in **both** files!

Reply:
mixxx-usb-uaccess.rules is still there.
It schould only not be installed to etc first. You mixxx.spec relies on that.
I have added a flag to control this.

Reply:
Cool, thank you :+1: